### ISSUE TITLE
Killer Is Dead crashes with d9vk

### ISSUE THREAD

**Issue by afettouhi** [Joshua-Ashton/d9vk#258]
### Software information
I am playing Killer is Dead on Steam using Proton 4.2-9 on Arch Linux 64 bit and KDE Plasma 5.16.3. The game works fine with this configuration but there are missing textures everywhere sadly. I read on protondb that d9vk fixes that issue. I installed d9vk 0.13f into the wine prefix that steam play recreates when launching the game initially. Now the missing textures are gone but the game crashes when loading missions that have cutscenes. This is the crash output from the terminal when launching steam via the terminal:

### System information
- GPU: Nvidia GTX 1080
- Driver: 430.34-1
- Wine version: Proton 4.2-9
- D9VK version: 0.13f

006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
Unhandled exception: 0x00000001 in 32-bit code (0x7b43d65d).
006f:fixme:dbghelp:elf_search_auxv can't find symbol in module
Register dump:
 CS:0023 SS:002b DS:002b ES:002b FS:0063 GS:006b
 EIP:7b43d65d ESP:0b18ef34 EBP:0b18efa8 EFLAGS:00000212(   - --  I   -A- - )
 EAX:7b42a451 EBX:7b640ff4 ECX:0b18efc0 EDX:7b43d5e0
 ESI:00000000 EDI:0b18f804
Stack dump:
0x0b18ef34:  f7cb943d 00000000 00000400 00000001
0x0b18ef44:  00000000 00000000 7b43d65d 00000000
0x0b18ef54:  00000000 00000000 00000000 00000000
0x0b18ef64:  5502de50 0b18ef90 04202c48 0b18efe0
0x0b18ef74:  00000400 00003fe2 040df882 04202c48
0x0b18ef84:  0b18f804 007052bf 040df882 00003fe2
Backtrace:
=>0 0x7b43d65d RaiseException+0x7d() in kernel32 (0x0b18efa8)
  1 0x007b5a71 EntryPoint+0xffffffff() in kidgame (0x0b18f804)
  2 0x0054004e EntryPoint+0xffffffff() in kidgame (0x0049003f)
0x7b43d65d RaiseException+0x7d in kernel32: addl        $12,%esp
Modules:
Module  Address                 Debug info      Name (222 modules)
PE        230000-  37b000       Deferred        wxmsw28u_vc_custom
PE        380000-  3d6000       Deferred        wxmsw28u_aui_vc_custom
PE        3e0000-  3e7000       Deferred        x3daudio1_7
PE        3f0000-  3f9000       Deferred        libogg
PE        400000- 4637000       Export          kidgame
PE       4b10000- 4ba5000       Deferred        wxmsw28u_xrc_vc_custom
PE       4bb0000- 4c75000       Deferred        wxmsw28u_richtext_vc_custom
PE       4c80000- 4d36000       Deferred        wxmsw28u_adv_vc_custom
PE       4d40000- 4dc0000       Deferred        wxmsw28u_html_vc_custom
PE       4dc0000- 4de2000       Deferred        wxmsw28u_xml_vc_custom
PE       4df0000- 4ff7000       Deferred        d3dcompiler_43
PE       5000000- 51ff000       Deferred        d3dx9_43
PE       5200000- 523f000       Deferred        d3dx11_43
PE       5240000- 5254000       Deferred        xapofx1_5
PE       5260000- 527c000       Deferred        nvtt
PE       5280000- 540f000       Deferred        libvorbis
PE       5410000- 541b000       Deferred        libvorbisfile
PE       5420000- 5467000       Deferred        binkw32
PE       5470000- 5499000       Deferred        physxextensions
PE       54a0000- 55ef000       Deferred        apexframework_x86
PE       55f0000- 5609000       Deferred        easyhook32
PE       5610000- 5629000       Deferred        libresample
PE       7570000- 79de000       Deferred        mscorlib
PE       c6d0000- c80d000       Deferred        steam
PE       d340000- d356000       Deferred        libpad
PE       d950000- d9c9000       Deferred        openvr_api_dxvk
PE       fe20000- fe33000       Deferred        physxloader
PE       ff50000- ffe6000       Deferred        physxcooking
PE      10000000-10327000       Deferred        wxmsw28u_core_vc_custom
PE      10920000-10cba000       Deferred        physxcore
PE      10cc0000-10d0a000       Deferred        cudart32_30_9
PE      10f50000-10fde000       Deferred        xaudio2_7
PE      16b80000-16cc5000       Deferred        apex_destructible_x86
PE      16de0000-16eed000       Deferred        apex_clothing_x86
PE      17000000-1704c000       Deferred        apex_framework_legacy_x86
PE      17160000-17212000       Deferred        apex_clothing_legacy_x86
PE      17340000-173cb000       Deferred        apex_destructible_legacy_x86
PE      1adf0000-1ae62000       Deferred        fonixttsdtsimpleus
PE      3b400000-3b41e000       Deferred        steam_api
PE      62440000-62696000       Deferred        d3d9
PE      6a540000-6a7da000       Deferred        d3d11
PE      6c540000-6dfa0000       Deferred        libmono-2.0-x86
PE      6f400000-6f5ba000       Deferred        dxgi
PE      78050000-780b9000       Deferred        msvcp100
PE      78aa0000-78b5f000       Deferred        msvcr100
ELF     7a800000-7a942000       Deferred        opengl32<elf>
  \-PE  7a820000-7a942000       \               opengl32
ELF     7b400000-7b7fc000       Export          kernel32<elf>
  \-PE  7b420000-7b7fc000       \               kernel32
ELF     7bc00000-7bd1c000       Deferred        ntdll<elf>
  \-PE  7bc10000-7bd1c000       \               ntdll
ELF     7c000000-7c004000       Deferred        <wine-loader>
ELF     7c2df000-7c33b000       Deferred        libvulkan.so.1
ELF     7c33b000-7c3c6000       Deferred        libgmp.so.10
ELF     7c3c6000-7c400000       Deferred        libhogweed.so.5
ELF     7c40c000-7c415000       Deferred        libffi.so.6
ELF     7c415000-7c453000       Deferred        libnettle.so.7
ELF     7c453000-7c468000       Deferred        libtasn1.so.6
ELF     7c468000-7c5aa000       Deferred        libp11-kit.so.0
ELF     7c5aa000-7c7b7000       Deferred        libgnutls.so.30
ELF     7c7b7000-7c848000       Deferred        libcups.so.2
ELF     7c848000-7c880000       Deferred        uxtheme<elf>
  \-PE  7c850000-7c880000       \               uxtheme
ELF     7c882000-7c88a000       Deferred        libxfixes.so.3
ELF     7c88a000-7c897000       Deferred        libxcursor.so.1
ELF     7c897000-7c8a5000       Deferred        libxrender.so.1
ELF     7c8a5000-7c8b9000       Deferred        libxi.so.6
ELF     7c8b9000-7c8be000       Deferred        libxcomposite.so.1
ELF     7c8be000-7c8c5000       Deferred        libxxf86vm.so.1
ELF     7c8c5000-7c8ca000       Deferred        libxinerama.so.1
ELF     7c9ca000-7ca5f000       Deferred        winex11<elf>
  \-PE  7c9d0000-7ca5f000       \               winex11
ELF     7cc37000-7cc63000       Deferred        libexpat.so.1
ELF     7cc63000-7ccaf000       Deferred        libfontconfig.so.1
ELF     7ccaf000-7cd28000       Deferred        libpcre.so.1
ELF     7cd28000-7ce60000       Deferred        libglib-2.0.so.0
ELF     7ce60000-7cf67000       Deferred        libharfbuzz.so.0
ELF     7cf67000-7cfa7000       Deferred        libpng16.so.16
ELF     7cfa7000-7cfb9000       Deferred        libbz2.so.1.0
ELF     7cfb9000-7d08e000       Deferred        libfreetype.so.6
ELF     7d08e000-7d0c7000       Deferred        mscoree<elf>
  \-PE  7d090000-7d0c7000       \               mscoree
ELF     7d0c7000-7d0de000       Deferred        powrprof<elf>
  \-PE  7d0d0000-7d0de000       \               powrprof
ELF     7d0de000-7d0f2000       Deferred        psapi<elf>
  \-PE  7d0e0000-7d0f2000       \               psapi
ELF     7d0f2000-7d115000       Deferred        bcrypt<elf>
  \-PE  7d100000-7d115000       \               bcrypt
ELF     7d115000-7d1e7000       Deferred        crypt32<elf>
  \-PE  7d120000-7d1e7000       \               crypt32
ELF     7d1e7000-7d232000       Deferred        dsound<elf>
  \-PE  7d1f0000-7d232000       \               dsound
ELF     7d232000-7d28e000       Deferred        dbghelp<elf>
  \-PE  7d240000-7d28e000       \               dbghelp
ELF     7d28e000-7d2b8000       Deferred        iphlpapi<elf>
  \-PE  7d2a0000-7d2b8000       \               iphlpapi
ELF     7d2b8000-7d2d2000       Deferred        wsock32<elf>
  \-PE  7d2c0000-7d2d2000       \               wsock32
ELF     7d2d2000-7d38d000       Deferred        msvcrt<elf>
  \-PE  7d2f0000-7d38d000       \               msvcrt
ELF     7d38d000-7d3d3000       Deferred        winevulkan<elf>
  \-PE  7d3a0000-7d3d3000       \               winevulkan
ELF     7d3d3000-7d3eb000       Deferred        vulkan-1<elf>
  \-PE  7d3e0000-7d3eb000       \               vulkan-1
ELF     7d3eb000-7d422000       Deferred        ws2_32<elf>
  \-PE  7d3f0000-7d422000       \               ws2_32
ELF     7d422000-7d44b000       Deferred        mpr<elf>
  \-PE  7d430000-7d44b000       \               mpr
ELF     7d44b000-7d465000       Deferred        libz.so.1
ELF     7d465000-7d4e2000       Deferred        wininet<elf>
  \-PE  7d470000-7d4e2000       \               wininet
ELF     7d4e2000-7d4f9000       Deferred        xinput1_3<elf>
  \-PE  7d4f0000-7d4f9000       \               xinput1_3
ELF     7d4f9000-7d624000       Deferred        libsdl2-2.0.so.0
ELF     7d624000-7d679000       Deferred        dinput8<elf>
  \-PE  7d630000-7d679000       \               dinput8
ELF     7d679000-7d6a5000       Deferred        msacm32<elf>
  \-PE  7d680000-7d6a5000       \               msacm32
ELF     7d6a5000-7d75e000       Deferred        winmm<elf>
  \-PE  7d6b0000-7d75e000       \               winmm
ELF     7d75e000-7d886000       Deferred        oleaut32<elf>
  \-PE  7d770000-7d886000       \               oleaut32
ELF     7d886000-7d8c6000       Deferred        winspool<elf>
  \-PE  7d890000-7d8c6000       \               winspool
ELF     7d8c6000-7d9bd000       Deferred        comdlg32<elf>
  \-PE  7d8d0000-7d9bd000       \               comdlg32
ELF     7d9bd000-7d9e2000       Deferred        imm32<elf>
  \-PE  7d9c0000-7d9e2000       \               imm32
ELF     7d9e2000-7da27000       Deferred        usp10<elf>
  \-PE  7d9f0000-7da27000       \               usp10
ELF     7da27000-7db64000       Deferred        comctl32<elf>
  \-PE  7da30000-7db64000       \               comctl32
ELF     7db64000-7dbe9000       Deferred        rpcrt4<elf>
  \-PE  7db70000-7dbe9000       \               rpcrt4
ELF     7dbe9000-7dd42000       Deferred        ole32<elf>
  \-PE  7dc00000-7dd42000       \               ole32
ELF     7dd42000-7dd66000       Deferred        shcore<elf>
  \-PE  7dd50000-7dd66000       \               shcore
ELF     7dd66000-7dd80000       Deferred        version<elf>
  \-PE  7dd70000-7dd80000       \               version
ELF     7dd80000-7deaf000       Deferred        gdi32<elf>
  \-PE  7dd90000-7deaf000       \               gdi32
ELF     7deaf000-7e0b0000       Deferred        user32<elf>
  \-PE  7dec0000-7e0b0000       \               user32
ELF     7e0b0000-7e11e000       Deferred        shlwapi<elf>
  \-PE  7e0c0000-7e11e000       \               shlwapi
ELF     7e11e000-7eadd000       Deferred        shell32<elf>
  \-PE  7e130000-7eadd000       \               shell32
ELF     7eadd000-7eb55000       Deferred        advapi32<elf>
  \-PE  7eaf0000-7eb55000       \               advapi32
ELF     7efec000-7f000000       Deferred        libnss_files.so.2
ELF     ed8d4000-ee734000       Deferred        libnvidia-glvkspirv.so.430.34
ELF     ef5c6000-ef719000       Deferred        wined3d<elf>
  \-PE  ef5e0000-ef719000       \               wined3d
ELF     ef973000-ef9ad000       Deferred        wbemprox<elf>
  \-PE  ef980000-ef9ad000       \               wbemprox
ELF     ef9ad000-efa74000       Deferred        msvcr80<elf>
  \-PE  ef9c0000-efa74000       \               msvcr80
ELF     efa74000-efb00000       Deferred        libvorbisenc.so.2
ELF     efc0d000-efc80000       Deferred        libflac.so.8
ELF     efe01000-efe11000       Deferred        libudev.so.0
ELF     efe11000-efe34000       Deferred        libgpg-error.so.0
ELF     eff3d000-eff52000       Deferred        avrt<elf>
  \-PE  eff40000-eff52000       \               avrt
ELF     eff52000-eff6b000       Deferred        libresolv.so.2
ELF     eff6b000-eff99000       Deferred        libvorbis.so.0
ELF     eff99000-f0078000       Deferred        libgcrypt.so.20
ELF     f0180000-f0189000       Deferred        libogg.so.0
ELF     f0189000-f0218000       Deferred        libsndfile.so.1
ELF     f0218000-f02cb000       Deferred        libsystemd.so.0
ELF     f02cb000-f0315000       Deferred        libdbus-1.so.3
ELF     f0315000-f03a4000       Deferred        libpulsecommon-12.2.so
ELF     f03a4000-f0400000       Deferred        libpulse.so.0
ELF     f0501000-f052d000       Deferred        liblzma.so.5
ELF     f052d000-f0556000       Deferred        winepulse<elf>
  \-PE  f0530000-f0556000       \               winepulse
ELF     f0556000-f0579000       Deferred        mmdevapi<elf>
  \-PE  f0560000-f0579000       \               mmdevapi
ELF     f097e000-f24fd000       Deferred        libnvidia-glcore.so.430.34
ELF     f2d08000-f2d10000       Deferred        libasyncns.so.0
ELF     f2d10000-f2dff000       Deferred        libglx_nvidia.so.0
ELF     f3014000-f308a000       Deferred        ddraw<elf>
  \-PE  f3020000-f308a000       \               ddraw
ELF     f308a000-f30a2000       Deferred        steamoverlayvulkanlayer.so
ELF     f30a2000-f30ea000       Deferred        libvklayer_steam_fossilize.so
ELF     f30ea000-f30fe000       Deferred        api-ms-win-core-localization-l1-2-1<elf>
  \-PE  f30f0000-f30fe000       \               api-ms-win-core-localization-l1-2-1
ELF     f3609000-f361d000       Deferred        api-ms-win-core-fibers-l1-1-1<elf>
  \-PE  f3610000-f361d000       \               api-ms-win-core-fibers-l1-1-1
ELF     f361d000-f3631000       Deferred        api-ms-win-core-synch-l1-2-0<elf>
  \-PE  f3620000-f3631000       \               api-ms-win-core-synch-l1-2-0
ELF     f3631000-f369e000       Deferred        setupapi<elf>
  \-PE  f3640000-f369e000       \               setupapi
ELF     f3903000-f3908000       Deferred        libnvidia-tls.so.430.34
ELF     f3908000-f392d000       Deferred        gameux<elf>
  \-PE  f3910000-f392d000       \               gameux
ELF     f392d000-f3946000       Deferred        hid<elf>
  \-PE  f3930000-f3946000       \               hid
ELF     f3a98000-f3b00000       Deferred        crashhandler.so
ELF     f4424000-f5d6c000       Deferred        steamclient.so
ELF     f5d6c000-f604c000       Deferred        lsteamclient<elf>
  \-PE  f5e70000-f604c000       \               lsteamclient
ELF     f6092000-f60b0000       Deferred        fusion<elf>
  \-PE  f60a0000-f60b0000       \               fusion
ELF     f60b5000-f60bf000       Deferred        libuuid.so.1
ELF     f7619000-f7621000       Deferred        libxdmcp.so.6
ELF     f7623000-f7628000       Deferred        libxau.so.6
ELF     f7628000-f7654000       Deferred        libxcb.so.1
ELF     f7654000-f7671000       Deferred        libgcc_s.so.1
ELF     f7671000-f76d3000       Deferred        libgldispatch.so.0
ELF     f76d3000-f76e9000       Deferred        libxext.so.6
ELF     f76e9000-f7836000       Deferred        libx11.so.6
ELF     f7836000-f7859000       Deferred        libglx.so.0
ELF     f7859000-f7925000       Deferred        libm.so.6
ELF     f7b00000-f7b06000       Deferred        libdl.so.2
ELF     f7b08000-f7b12000       Deferred        librt.so.1
ELF     f7b12000-f7b76000       Deferred        libgl.so.1
ELF     f7b76000-f7d52000       Deferred        libc.so.6
ELF     f7d52000-f7d73000       Deferred        libpthread.so.0
ELF     f7d73000-f7f30000       Export          libwine.so.1
ELF     f7f30000-f7f6e000       Deferred        gameoverlayrenderer.so
ELF     f7f70000-f7f9a000       Deferred        ld-linux.so.2
Threads:
process  tid      prio (all id:s are in hex)
00000008 steam.exe
        00000028    0
        00000009    0
0000000e services.exe
        00000024    0
        00000021    0
        0000001c    0
        00000018    0
        00000013    0
        00000010    0
        0000000f    0
00000011 winedevice.exe
        00000019    0
        00000017    0
        00000016    0
        00000012    0
0000001a plugplay.exe
        0000001e    0
        0000001d    0
        0000001b    0
0000001f winedevice.exe
        00000027    0
        00000026    0
        00000025    0
        00000023    0
        00000022    0
        00000020    0
00000029 explorer.exe
        0000002f    0
        0000002e    0
        0000002d    0
        0000002a    0
0000002b (D) Z:\home\af\.local\share\Steam\steamapps\common\KillerIsDead\Binaries\Win32\KidGame.exe
        0000006d    1
        00000067    0
        00000066    0
        00000065    2
        00000064    2
        00000063    2
        00000062    0
        00000061    0
        00000060    0
        0000005f    0
        0000005e    0
        0000005d    0
        0000005c    0
        0000005b    0
        0000005a    0
        00000059    0
        00000058    0
        00000057   -2
        00000056   -2
        00000055   -2
        00000054   -2
        00000053    0
        00000052    0
        00000050    0
        0000004f   15
        0000004e   15
        0000004d    0
        0000004c    0
        00000042    0
        00000041    0
        00000040    0
        0000003f    0
        0000003e    0
        0000003d    0
        0000003c    0
        0000003b    0 <==
        0000003a    0
        00000039    0
        00000038    0
        00000037    0
        00000036    0
        00000035    0
        00000034    0
        00000033    0
        00000032    0
        00000031    0
        00000030    0
        0000002c    0
System information:
    Wine build: wine-4.2
    Platform: i386 (WOW64)
    Version: Windows 7
    Host system: Linux
    Host version: 5.2.1-arch1-1-ARCH
JS method call Messaging.PostMessage with 4 arguments
pid 17742 != 17741, skipping destruction (fork without exec?)
Game removed: AppID 261110 "", ProcID 17782 
Exiting app 261110
JS method call WebChat.GetOverlayChatBrowserInfo with 1 arguments
JS method call Storage.SetString with 3 arguments
JS method call Messaging.PostMessage with 4 arguments
JS method call Messaging.PostMessage with 4 arguments
OnFocusWindowChanged to window type: k_nGameIDControllerConfigs_Desktop, AppID 413080
Loaded Config for Local Selection Path for App ID 413080, Controller 0: /home/af/.local/share/Steam/steamapps/workshop/content/241100/650764041/controller_configuration.vdf
saving roaming config store to 'sharedconfig.vdf'
roaming config store 2 saved successfully
Scheduled disconnect of controller 0
Local Device Found
  type: 28de 1142
  path: /dev/hidraw0
  serial_number:  - 0
  Manufacturer: Valve Software
  Product:      Steam Controller
  Release:      1
  Interface:    0

Controller 0 disconnected
Local Device Found
  type: 28de 1142
  path: /dev/hidraw0
  serial_number:  - 0
  Manufacturer: Valve Software
  Product:      Steam Controller
  Release:      1
  Interface:    0

Installing breakpad exception handler for appid(steam)/version(1563411792)
Installing breakpad exception handler for appid(steam)/version(1563411792)
Installing breakpad exception handler for appid(steam)/version(1563411792)

**Comment by Joshua-Ashton** [comment]
Can you produce an API trace on WineD3D that has the issue?

**Comment by afettouhi** [comment]
I could but I have no idea on how to do that. Could you provide some documentation on how to do that maybe?

**Comment by Joshua-Ashton** [comment]
There is a guide on https://tracehub.froggi.es

**Comment by afettouhi** [comment]
The link doesn't work. Website seems to be down or something.

**Comment by afettouhi** [comment]
I can see on the original dxvk github page there is a section on apitrace:

### Apitrace
If you're opening an issue and you're having issues with making an apitrace, put d3d11.dll, dxgi.dll 
and dxgitrace.dll from your apitrace build into the directory where yourgame.exe is located. This will 
create a file yourgame.trace without requiring further interaction.

Does d9vk has this feature as well?

**Comment by Joshua-Ashton** [comment]
> The link doesn't work. Website seems to be down or something.

Should be back up now.

Yes, (but its not a d9vk/dxvk feature,) just `s/d3d11/d3d9` and ignore dxgi.

**Comment by afettouhi** [comment]
So I then just copy d3d9.dll and dxgitrace.dll into the game folder then?

**Comment by Joshua-Ashton** [comment]
just d3d9.dll

**Comment by afettouhi** [comment]
The dxgitrace.dll isn't needed for the trace?

**Comment by Joshua-Ashton** [comment]
no

**Comment by afettouhi** [comment]
I just tried making an apitrace. I put the d3d9.dll folder where the executable is but no trace file is present after the game crashes as I previously explained.

**Comment by Joshua-Ashton** [comment]
Should be on the desktop or the desktop of your wine prefix

Might be a good idea to make the trace with wined3d also.if it crashes

**Comment by afettouhi** [comment]
Found it in the desktop folder of my prefix. But the file is over 420 MB large. Is there a way to compress it down to a more usable form?

**Comment by Joshua-Ashton** [comment]
lzma

**Comment by afettouhi** [comment]
The file is still several hundred mb after compression. So I can't upload it.

**Comment by Joshua-Ashton** [comment]
> The file is still several hundred mb after compression. So I can't upload it.

I can't debug the issue without it

**Comment by afettouhi** [comment]
I am aware of that. Any suggestion to where I can upload it so that you can pull it down.

**Comment by Joshua-Ashton** [comment]
Mega? Google Drive? https://git.froggi.es?

**Comment by afettouhi** [comment]
Hopefully this works. Uploaded to google drive should be able to access it via this link

https://drive.google.com/file/d/1AZ4gabdXDwROmTWnelz9dhdadkoz8bJQ/view?usp=sharing

**Comment by Joshua-Ashton** [comment]
The trace doesn't crash for me. Did you record the trace with D9VK or WineD3D?

**Comment by afettouhi** [comment]
With d9vk.

**Comment by Joshua-Ashton** [comment]
Can you try and make a trace with WineD3D instead?

**Comment by afettouhi** [comment]
Sure.

**Comment by afettouhi** [comment]
Here is the apitrace loading the same level with wined3d. Only difference is that level is loaded hence no crash and I just quit the game normally after that.
[KifGame_WINED3D.zip](https://github.com/Joshua-Ashton/d9vk/files/3414793/KifGame_WINED3D.zip)

**Comment by Joshua-Ashton** [comment]
These traces are invalid.

**Comment by afettouhi** [comment]
OK, that I don't understand. I did the same thing as before when I traced the with d9vk. Only difference here is that d9vk isn't installed this time.

**Comment by Joshua-Ashton** [comment]
Can you try it again? A 3KB trace wont give me anything...

**Comment by afettouhi** [comment]
For some reason I can only get it to produce this 3kB trace or none at all. I am going try to redownload the game again after deleting all the directories associated with it.

**Comment by afettouhi** [comment]
Here is a new trace (zipfile) with d9vk again from fresh install of the game. Same level is loaded and the crash happens roughly at the same place in the cutscene.

https://drive.google.com/file/d/1ckml-96ZfY2-GXYt0yO_iD1uAN1jD7ZE/view?usp=sharing

I can't get the game to produce a trace with wined3d. Do I need to do something else with wined3d in order to get an apitrace besides not installing d9vk?

**Comment by Joshua-Ashton** [comment]
> Do I need to do something else with wined3d in order to get an apitrace besides not installing d9vk?

Nope. Not sure what the issue is here :-/

**Comment by Joshua-Ashton** [comment]
Do you know if d9vk ever used to work with this game?

**Comment by afettouhi** [comment]
No, I don't. I have only tried the game with proton, hence wined3d. I just installed d9vk because one of the later protondb reports for the game said that d9vk fixed the missing texture issue that the game has had for a long time in wine.

**Comment by afettouhi** [comment]
No for it launches in directx 9 mode. I haven't seen an option to switch to dx10/11 anywhere.

**Comment by arzardk** [comment]
@Joshua-Ashton first of all, here is an apitrace from Windows if it is helpful somehow: [Killer Is Dead (Steam 261110) (d3d9).tar.xz](https://drive.google.com/open?id=1ikvy7hd3yw6lrUZTwY1Ub6-yefLf3GNz)

I tried to replay this trace on my Ubuntu 19.04, but cannot do this because of the error linked to the unsupported API.
Also i tried to run this game through d9vk on Ubuntu and it does not launched because of the error mentioned below.
> 
> 
> No for it launches in DirectX 9 mode. I haven't seen an option to switch to dx10/11 anywhere.

You are half right, because i found something interesting about how this game works with APIs, though I'm not sure in rightfulness of my conclusions:

1. I launched the game with d3d9.dll but without dxgi.dll - game launched fine and rendered correctly
[KidGame_d3d9.log](https://gist.github.com/arzardk/e8df26d65e2eddb53cbc9d6c77a006c6)
2. I launched the game with d3d9.dll and with dxgi.dll, here is the result
![kid_crash](https://user-images.githubusercontent.com/37209942/61686952-c7459000-ad29-11e9-8205-96d2b67c95f2.png)
[KidGame_dxgi.log](https://gist.github.com/arzardk/ae23c7d58031dfe934a9ba1d9e058fe4)
[KidGame_d3d9.log](https://gist.github.com/arzardk/bb031c1031bd7131c2536eac27e65780)
3. I launched the game with d3d9.dll, dxgi.dll and with d3d11.dll from d9vk - game launched and rendered correctly.
[KidGame_d3d11.log](https://gist.github.com/arzardk/2a8b292826b2bb8da61ecba93bdb7e3a)
[KidGame_dxgi.log](https://gist.github.com/arzardk/2626d12baead248e18b835ca0b2d99a0)
[KidGame_d3d9.log](https://gist.github.com/arzardk/54d7d590502fc559da4bf0af9c88692f)

So i think this game is somehow using DirectX 9 and 11 (yeah, sounds ridiculous). I remember one game with similar behavior: The Evil Within, which using d3d11 and opengl render, according to information from GeForce forum.

**Comment by NerosTie** [comment]
@Joshua-Ashton 

wined3d:
https://mega.nz/#!vmBH2aCQ!yfp4sDfcBykj3C9bGEzslIXdbmgq68PJAkSrq7CJWRE
[steam-261110-wined3d.log](https://github.com/Joshua-Ashton/d9vk/files/3435411/steam-261110-wined3d.log)

d9vk:
https://mega.nz/#!vmBH2aCQ!yfp4sDfcBykj3C9bGEzslIXdbmgq68PJAkSrq7CJWRE
[steam-261110-d9vk.log](https://github.com/Joshua-Ashton/d9vk/files/3435413/steam-261110-d9vk.log)

GPU: Nvidia GT 730M
Driver: 430.26-9
Wine version: Proton 4.2
DXVK version: #725 (jobs)

**Comment by Joshua-Ashton** [comment]
> I launched the game with d3d9.dll and with dxgi.dll, here is the result [...] I launched the game with d3d9.dll, dxgi.dll and with d3d11.dll from d9vk - game launched and rendered correctly.

Ok, so just install everything so that things work together nicely? I don't get why this is my issue that you frogged up your install one time. 🐸

You can't just choose and pick random DLLs, there are certain ones that won't work in certain scenarios.

You can use d3d9.dll independently, but you can't use something that depends on d3d11.dll without dxgi.dll (and vice versa) without getting issues.

**Comment by arzardk** [comment]
@Joshua-Ashton these DLL's are not random, they are from d9vk 0.13f release. And no, this is not your issue, sorry.

**Comment by afettouhi** [comment]
Just tested this again using Proton 4.11-1 with the `PROTON_USE_D9VK=1 %command%` turned on and I get the same crash. Without the game runs fine plus the missing texture issue is gone, so it seems to fixed in proton now.

**Comment by arzardk** [comment]
> Just tested this again using Proton 4.11-1 with the `PROTON_USE_D9VK=1 %command%` turned on and I get the same crash. Without the game runs fine plus the missing texture issue is gone, so it seems to fixed in proton now.

What GPU and mesa version are you using? I've tested this game today with `PROTON_USE_D9VK=1 %command%` and it works fine, except for some artifacts when you're dating girls.
My GPU is Vega 56 and I'm using Mesa+ACO driver.

**Comment by afettouhi** [comment]
Mesa 19.1.3 on Arch Linux with a Nvidia GTX 1080 using driver 430.40.

**Comment by arzardk** [comment]
> Mesa 19.1.3 on Arch Linux with a Nvidia GTX 1080 using driver 430.40.

Seems like this issue is nVidia specific. Need some tests from Radeon users to be sure though.

**Comment by FreedomFaighter** [comment]
Having 0028:fixme:dbghelp:elf_search_auxv can't find symbol in module when trying to login to certain zones in tlp everquest, didn't create a new issue as it might be related

**Comment by NerosTie** [comment]
Tested with Proton-tkg 5.18 + WINED3D and it crashes too (during the cutscene with the sword). It doesn't crash with Proton 5.0-9 + WINED3D.... so it looks like with newer versions of Wine, WINED3D is affected with the same issue as Proton-X + DXVK.

(btw, I don't have sound with Proton-tkg, if you know why, tell me)
